spec/features/user_writes_a_review_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.feature "User writes a review", type: :feature do
+  scenario "if he has written no previous review for a book" do
+    create(
+      :book,
+      title: "The Shades: a history",
+      summary: "Follow the author around Ankh-Morpork's finest gastro pubs",
```

Comment:
I think we don't need to specify the summary and cover for this particular spec.